.github/workflows/run_tutorials_v1.yml

```diff
@@ -0,0 +1,140 @@
+name: Run Tutorials for v1.x
+
+on:
+  workflow_dispatch: # Activate this workflow manually
+  pull_request:
+    paths:
+      # This workflow must run only for v1.x tutorials
+      # Some v1 tutorials are ignored in any case as we can't run those
+      # for different reasons
+      - "tutorials/01_Basic_QA_Pipeline.ipynb"
+      - "tutorials/03_Scalable_QA_System.ipynb"
+      - "tutorials/04_FAQ_style_QA.ipynb"
+      - "tutorials/05_Evaluation.ipynb"
+      - "tutorials/06_Better_Retrieval_via_Embedding_Retrieval.ipynb"
+      - "tutorials/08_Preprocessing.ipynb"
+      - "tutorials/11_Pipelines.ipynb"
+      - "tutorials/15_TableQA.ipynb"
+      - "tutorials/16_Document_Classifier_at_Index_Time.ipynb"
+      - "tutorials/17_Audio.ipynb"
+      - "tutorials/19_Text_to_Image_search_pipeline_with_MultiModal_Retriever.ipynb"
+      - "tutorials/20_Using_Haystack_with_REST_API.ipynb"
+      - "tutorials/21_Customizing_PromptNode.ipynb"
+      - "tutorials/26_Hybrid_Retrieval.ipynb"
```

Comment:
There were some tutorials that we skipped due to the lack of API Keys. I added them + the ones that have a pinned haystack version now that we have support for it as well
```suggestion
      - "tutorials/01_Basic_QA_Pipeline.ipynb"
      - "tutorials/03_Scalable_QA_System.ipynb"
      - "tutorials/04_FAQ_style_QA.ipynb"
      - "tutorials/05_Evaluation.ipynb"
      - "tutorials/06_Better_Retrieval_via_Embedding_Retrieval.ipynb"
      - "tutorials/07_RAG_Generator.ipynb"
      - "tutorials/08_Preprocessing.ipynb"
      - "tutorials/10_Knowledge_Graph.ipynb"
      - "tutorials/11_Pipelines.ipynb"
      - "tutorials/14_Query_Classifier.ipynb"
      - "tutorials/15_TableQA.ipynb"
      - "tutorials/16_Document_Classifier_at_Index_Time.ipynb"
      - "tutorials/17_Audio.ipynb"
      - "tutorials/19_Text_to_Image_search_pipeline_with_MultiModal_Retriever.ipynb"
      - "tutorials/20_Using_Haystack_with_REST_API.ipynb"
      - "tutorials/21_Customizing_PromptNode.ipynb"
      - "tutorials/22_Pipeline_with_PromptNode.ipynb"
      - "tutorials/23_Answering_Multihop_Questions_with_Agents.ipynb"
      - "tutorials/24_Building_Chat_App.ipynb"
      - "tutorials/25_Customizing_Agents.ipynb"
      - "tutorials/26_Hybrid_Retrieval.ipynb"
```